app/renderer/components/preferences/extensionsTab.js

```diff
@@ -76,27 +76,16 @@ class ExtensionsTab extends ImmutableComponent {
           settings={this.props.settings}
           checked={this.getCheckedExtension(extension.get('id'))}
           onChangeSetting={this.props.onChangeSetting} />
-      },
-      { // Exclude option
-        /* TODO @cezaraugusto reenable it once we can be able
-         * to recover from an excluded->re-enabled extension state
-        html: !extension.get('isDummy') && !isBuiltInExtension(extension.get('id'))
-        ? <div className={globalStyles.appIcons.trash}
-          onClick={this.onRemoveExtension.bind(this, extension.get('id'))} />
-        : <span data-l10n-id={isBuiltInExtension(extension.get('id')) ? 'integrated' : 'notInstalled'} />
-        */
       }
-    ]
-  }
-
-  get columnClassNames () {
-    return [
-      css(styles.tableRow__column, styles.tableRow__column_center),
-      css(styles.tableRow__column),
-      css(styles.tableRow__column),
-      css(styles.tableRow__column),
-      css(styles.tableRow__column, styles.tableRow__column_center)
-      // css(styles.tableRow__column, styles.tableRow__column_center)
+      // { // Exclude option
+      //  /* TODO @cezaraugusto reenable it once we can be able
+      //   * to recover from an excluded->re-enabled extension state
+      //  html: !extension.get('isDummy') && !isBuiltInExtension(extension.get('id'))
+      //  ? <div className={globalStyles.appIcons.trash}
+      //    onClick={this.onRemoveExtension.bind(this, extension.get('id'))} />
+      //  : <span data-l10n-id={isBuiltInExtension(extension.get('id')) ? 'integrated' : 'notInstalled'} />
+      //  */
+      // }
```

Comment:
This block is commented out not to display the most right column which otherwise would appear due to the global padding setting.

Reply:
thanks I'm going to exclude it